test/e2e/upgrade/alert/alert.go

```diff
@@ -153,6 +159,9 @@ count_over_time(ALERTS{alertstate="firing",severity!="info",alertname!~"Watchdog
 		violation := fmt.Sprintf("alert %s fired for %s seconds with labels: %s", series.Metric["alertname"], series.Value, helper.LabelsAsSelector(labels))
 		if cause := firingAlertsWithBugs.Matches(series); cause != nil {
 			knownViolations.Insert(fmt.Sprintf("%s (open bug: %s)", violation, cause.Text))
+		} else if cause = allowedFiringAlerts.Matches(series); cause != nil {
```

Comment:
No, please open a bug.  The alert is no different than any other alert.  You should be adding the bug that fixes this to the section above.